pages/services/edge-lb/1.2/release-notes/index.md

```diff
@@ -51,41 +94,6 @@ Released on September 11, 2018.
 * Bump `haproxy` from 1.8.12 to 1.8.13. [changelog] (http://git.haproxy.org/?p=haproxy-1.8.git;a=blob;f=CHANGELOG;h=aed48fc5fb951aff7dd458c4bc9bfcfe1d5dd99a;hb=HEAD)
 * Commit protobuf code changes that stem from [tool update](https://github.com/golang/protobuf/tree/master/protoc-gen-go).
 
-
-### Shortlist
-
-```
```

Comment:
We removed this because the quality of this information is not the best.

Reply:
Make sure to have the Shortlist saved somewhere in case we need it in the future.